### ISSUE TITLE
Changing git branch in IDE isn't reflected in the PS1 in the console

### ISSUE THREAD

**Issue by sebhewelt** [robbyrussell/oh-my-zsh#6711]
As Is:
Changing git branch in IDE isn't reflected in the PS1 in the console. The branch is indeed switched, because when i do `gco branch-name` it logs `Already on branch-name`, but it was only after running that command.
TO BE:
 I wish the CL would refresh on changing branch in my editor (VS code)

🙏

**Comment by mcornella** [comment]
This is already done by some themes, you need to find one that does it or modify the one you're already using (see [Customization](https://github.com/robbyrussell/oh-my-zsh/wiki/Customization#overriding-and-adding-themes)).
